k8s/tools/deploy-pubsub-gke.sh

```diff
@@ -0,0 +1,262 @@
+if [[ "$*" != *--no-datastore* ]] ; then
+  echo "Enabling Datastore API and deploying datastore index"
+  gcloud --project $DEVSHELL_PROJECT_ID -q services enable datastore.googleapis.com
+  gcloud --project $DEVSHELL_PROJECT_ID -q datastore indexes create ../tools/gcf_init/index.yaml
+fi
+
+# Create GKE cluster and authenticate to it
+echo "Enabling GCP Compute and Container APIs"
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable compute.googleapis.com
+gcloud -q services --project $DEVSHELL_PROJECT_ID enable container.googleapis.com
+echo "Creating cluser $CLUSTER_NAME with $CLUSTER_NODE_SIZE node(s) configured with machine type $CLUSTER_MACHINE_TYPE and disk size $CLUSTER_MACHINE_SIZE"
+gcloud beta container clusters create $CLUSTER_NAME --machine-type $CLUSTER_MACHINE_TYPE --disk-size $CLUSTER_MACHINE_SIZE --num-nodes $CLUSTER_NODE_SIZE --master-ipv4-cidr $VPC_CONTROL_PANE --network $VPC_NETWORK --zone $ZONE --shielded-secure-boot --no-enable-master-authorized-networks  --enable-private-nodes --enable-ip-alias  --scopes "https://www.googleapis.com/auth/cloud-platform" --labels "turbinia-infra=true"
+gcloud container clusters get-credentials $CLUSTER_NAME --zone $ZONE
+
+# Go to deployment folder to make changes files
+cd $DEPLOYMENT_FOLDER
+
+
+# Disable some jobs
+echo "Updating $TURBINIA_CONFIG with disabled jobs"
+sed -i -e "s/DISABLED_JOBS = .*/DISABLED_JOBS = $DISABLED_JOBS/g" $TURBINIA_CONFIG
+
+# Update Turbinia config with project info
+echo "Updating $TURBINIA_CONFIG config with project info"
+sed -i -e "s/INSTANCE_ID = .*/INSTANCE_ID = '$INSTANCE_ID'/g" $TURBINIA_CONFIG
+sed -i -e "s/TURBINIA_PROJECT = .*/TURBINIA_PROJECT = '$DEVSHELL_PROJECT_ID'/g" $TURBINIA_CONFIG
+sed -i -e "s/TURBINIA_ZONE = .*/TURBINIA_ZONE = '$ZONE'/g" $TURBINIA_CONFIG
+sed -i -e "s/TURBINIA_REGION = .*/TURBINIA_REGION = '$REGION'/g" $TURBINIA_CONFIG
+
+# Create File Store instance and update deployment files with created instance
+if [[ "$*" != *--no-filestore* ]] ; then  
+  echo "Enabling GCP Filestore API"
+  gcloud -q services --project $DEVSHELL_PROJECT_ID enable file.googleapis.com
+  echo "Creating Filestore instance $FILESTORE_NAME with capacity $FILESTORE_CAPACITY"
+  gcloud -q filestore --project $DEVSHELL_PROJECT_ID instances create $FILESTORE_NAME --file-share=name=$FILESTORE_NAME,capacity=$FILESTORE_CAPACITY --zone=$ZONE --network=name=$VPC_NETWORK
+else
+  echo "Using pre existing Filestore instance $FILESTORE_NAME with capacity $FILESTORE_CAPACITY"
+fi
+
+echo "Updating $TURBINIA_CONFIG config with Filestore configuration"
+FILESTORE_IP=$(gcloud filestore instances describe $FILESTORE_NAME --zone=$ZONE --format='value(networks.ipAddresses)' --flatten="networks[].ipAddresses[]")
+FILESTORE_MOUNT="'\/mnt\/$FILESTORE_NAME'"
+echo "Updating $TURBINIA_CONFIG config with Filestore configuration"
+sed -i -e "s/<IP_ADDRESS>/$FILESTORE_IP/g" turbinia-output-filestore.yaml
+sed -i -e "s/output/$FILESTORE_NAME/g" *.yaml
```

Comment:
This is a pretty open regex when just searching for `output` in all of the files.  Is there somethign more specific we can add to search for here?

Reply:
You might also consider adding start of line and end of line anchors to these where appropriate as well.

Reply:
Done